src/google/protobuf/stubs/atomicops.h

```diff
@@ -194,14 +194,6 @@ Atomic64 Release_Load(volatile const Atomic64* ptr);
 #elif defined(GOOGLE_PROTOBUF_OS_AIX)
 #include <google/protobuf/stubs/atomicops_internals_power.h>
 
-// Apple.
```

Comment:
I don't have access to an older mac, but the intrinsics work on clang/llvm 3.4 from 2014 (tested on my desktop). Looking at commits, should be fine for versions from 2011 forward. How old do we want to support?